examples/try.html

```diff
@@ -0,0 +1,1471 @@
+}, {
+  "example": [ {
+      "left" : "item 1 left",
+      "right" : "item 1 right"    
+    }, {
+      "left" : "item 2 left",
+      "right" : "item 2 right"            
+    }, {
+      "left" : "item 3 left",
+      "right" : "item 3 right"            
+    }
+  ],
+  "notes": "A list of objects produces one row for each object"
+}, {
+  "example": [{
+      "left" : "item 1",
+      "right" : [
+        "item 1.1",
+        "item 1.2"
+      ]
+    }, {
+      "left" : "item 2",
+      "right" : [{
+          "sub": {
+            "a": "item 2.1",
+            "b": [1, 2, 3]
+          }
+        }, {
+          "sub": {
+            "a": "item 2.2",
+            "b": ["a", "b", "c"]
+          }
+        }
+      ]
+    } 
+  ],
+  "notes": "List of objects with nested list"
+}, {
+  "example": {
+    "left" : [
+      "left item 1",
+      "left item 2",
+      "left item 3"
+    ],
+    "right" : [
+      "right item 1",
+      "right item 2"
+    ]
+  },
+  "notes": "Parallel lists are rendered side-by-side. The number of rows is the length of the longest list"
+}, {
+  "example": {
+    "left" : [
+      "left item 1",
+      "left item 2",
+      "left item 3"
+    ],
+    "right" : [
+      [
+        "right item 1-1",
+        "right item 1-2"
+      ], [
+        "right item 2-1",
+        "right item 2-2"
+      ]
+    ]
+  },
+  "notes": "Nested lists [[]] are flattened out"
+}, {
+  "example": {
+    "left": null
+  },
+  "notes": "Null values and missing fields produce empty cells."
+}, {
+  "example": {
+    "left" : {
+      "x": [
+        ["string", "ABC"],
+        ["number", 1.23],
+        ["bool", true],
+        ["null", null]
+      ]
+    }
+  },
+  "notes": "Tuples (heterogenous lists) are rendered side by side if they are in the deepest nesting. They can be in a list."
+}, {
+  "example": [
+    {
+      "left": 1
+    }, {
+      "right": 2
+    }
+  ],
+  "notes": "A tuple (list) of objects with disjunct keys is rendered side-by-side."
+}, {
+  "example": {
+    "left": {
+      "x": [47.492, 19.752]
+    }
+  },
+  "notes": "Two element lists are also tuples. Handy for latlng and such."
+}, {
+  "example": {
+    "left" : [
+      ["left item 1"],
+      [[["left item 2a", ["left item 2b"]], "left item 2c"]],
+      "left item 3"
+    ],
+    "right" : [
+      "right item 1",
+      {
+        "sub": [{
+          "a": "Just a",
+          "b": [0.1, 0.2, 0.3]
+        }]
+      }
+    ]
+  },
+  "notes": "Objects and lists can be nested arbitrarily, even mixing types."
+}
+]
+</pre>
+<pre id="example-readme">{
+  "userId": 8927524,
+  "profile": {
+    "name":   "Mary Jane",
+    "age":    29,
+    "gender": "female"
+  },
+  "comments": [{
+    "id":       "F2372BAC",
+    "text":     "I concur.",
+    "replyTo":  [9817361, "F8ACD164F"],
+    "time":     "2015-02-03"
+  }, {
+    "id":       "GH732AFC",
+    "replyTo":  [9654726, "A44124F"],
+    "time":     "2015-03-01"
+  }]
+}
```

Comment:
These are great! It would also be good to have a 3 tuple example, and maybe [this real-world example](https://dev.twitter.com/rest/reference/get/search/tweets).